### PR TITLE
Add Homebrew to install and note on code signing

### PR DESCRIPTION
**Pull request by DaveEveritt**
Many Mac users have the Homebrew package manager so added brew install to the guide, and a note on first launch about circumventing the code signing issue

### CODE REVIEW

File: docs/Introduction.md [hunk 1]
```diff
@@ -28,10 +28,22 @@ The latest version runs on macOS 10.10 or later. [Previous versions](https://qsa
...
 
+Using Homebrew:
+
+[Homebrew is a package manager for OS X](https://brew.sh) for installs, updates and uninstalls of most OS X software:
+
+Open Terminal, type `brew install --cask quicksilver`. You can `uninstall` just as easily. 
+
+## First Launch
+
+Due to the way Quicksilver is built, OS X raises a [code signing warning (open issue)](https://github.com/quicksilver/Quicksilver/issues/2576))—the quickest way to get around this is to <kbd>ctrl-</kbd> or <kbd>right-click</kbd> the Quicksilver app in Applications, choose "open" from the contextual menu, *then* choose the *extra "open" button* that now appears in the warning dialog box. This only needs doing once, Quicksilver will then launch in future without further issue.
```
Comment (didaquis) [inline docs/Introduction.md:45]: Hi. 

Maybe the `OS X` reference should be changed to `macOS` ??? 

https://en.wikipedia.org/wiki/MacOS

Comment (DaveEveritt) [reply docs/Introduction.md:45]: Agreed . Old habits! macOS it is, not OS X.
```suggestion
Due to the way Quicksilver is built, macOS raises a [code signing warning (open issue)](https://github.com/quicksilver/Quicksilver/issues/2576))—the quickest way to get around this is to <kbd>ctrl-</kbd> or <kbd>right-click</kbd> the Quicksilver app in Applications, choose "open" from the contextual menu, *then* choose the *extra "open" button* that now appears in the warning dialog box. This only needs doing once, Quicksilver will then launch in future without further issue.
```

**Review by DaveEveritt — COMMENTED** [review]
update "OS X" to "macOS"

**Comment by skurfer** [comment]
Thanks for the contributions here and on the main site!

**Pull request merged**